src/Type/User/Mutation/UserMutation.php

```diff
@@ -0,0 +1,291 @@
+<?php
+
+namespace WPGraphQL\Type\User\Mutation;
+
+use WPGraphQL\Types;
+
+/**
+ * Class UserMutation
+ *
+ * @package WPGraphQL\Type\User\Mutation
+ */
+class UserMutation {
+
+	/**
+	 * Stores the input fields static definition
+	 *
+	 * @var array $input_fields
+	 * @access private
+	 */
+	private static $input_fields = [];
+
+	/**
+	 * Defines the accepted input arguments
+	 *
+	 * @return array|null
+	 * @access public
+	 */
+	public static function input_fields() {
+
+		if ( empty( self::$input_fields ) ) {
+
+			$input_fields = [
+				'password'    => [
+					'type'        => Types::string(),
+					'description' => __( 'A string that contains the plain text password for the user.', 'wp-graphql' ),
+				],
+				'nicename'    => [
+					'type'        => Types::string(),
+					'description' => __( 'A string that contains a URL-friendly name for the user. The default is the user\'s username.', 'wp-graphql' ),
+				],
+				'websiteUrl'  => [
+					'type'        => Types::string(),
+					'description' => __( 'A string containing the user\'s URL for the user\'s web site.', 'wp-grapql' ),
+				],
+				'email'       => [
+					'type'        => Types::string(),
+					'description' => __( 'A string containing the user\'s email address.', 'wp-graphql' ),
+				],
+				'displayName' => [
+					'type'        => Types::string(),
+					'description' => __( 'A string that will be shown on the site. Defaults to user\'s username. It is likely that you will want to change this, for both appearance and security through obscurity (that is if you dont use and delete the default admin user).', 'wp-graphql' ),
+				],
+				'nickname'    => [
+					'type'        => Types::string(),
+					'description' => __( 'The user\'s nickname, defaults to the user\'s username.', 'wp-graphql' ),
+				],
+				'firstName'   => [
+					'type'        => Types::string(),
+					'description' => __( '	The user\'s first name.', 'wp-graphql' ),
+				],
+				'lastName'    => [
+					'type'        => Types::string(),
+					'description' => __( 'The user\'s last name.', 'wp-graphql' ),
+				],
+				'description' => [
+					'type'        => Types::string(),
+					'description' => __( 'A string containing content about the user.', 'wp-graphql' ),
+				],
+				'richEditing' => [
+					'type'        => Types::string(),
+					'description' => __( 'A string for whether to enable the rich editor or not. False if not empty.', 'wp-graphql' ),
+				],
+				'registered'  => [
+					'type'        => Types::string(),
+					'description' => __( 'The date the user registered. Format is Y-m-d H:i:s.', 'wp-graphql' ),
+				],
+				'roles'       => [
+					'type'        => Types::list_of( Types::string() ),
+					'description' => __( 'An array of roles to be assigned to the user.', 'wp-graphql' ),
+				],
+				'jabber'      => [
```

Comment:
AIM, YIM and JABBER were removed from core in 3.6 (https://developer.wordpress.org/reference/functions/wp_insert_user/#changelog) so not going to include these in the core user mutations.